test/runtests.jl

```diff
@@ -28,7 +28,13 @@ tests = (ARGS==["all"] || isempty(ARGS)) ? testnames : ARGS
 if "linalg" in tests
     # specifically selected case
     filter!(x -> x != "linalg", tests)
-    prepend!(tests, ["linalg1", "linalg2", "linalg3", "linalg4", "linalg/lapack", "linalg/triangular", "linalg/tridiag", "linalg/pinv", "linalg/cholmod", "linalg/umfpack", "linalg/givens"])
+    prepend!(tests, ["linalg1", "linalg2", "linalg3", "linalg4", "linalg/lapack", "linalg/triangular", "linalg/tridiag", "linalg/pinv", "linalg/givens"])
+end
+
+if "sparse" in tests
+    # specifically selected case
+    filter!(x -> x != "sparse", tests)
+    prepend!(tests, ["sparse/sparse", "sparse/cholmod", "sparse/umfpack"])
```

Comment:
This should probably be moved a few lines earlier to before the "linalg" version, so the linalg group gets prepended later and therefore run earlier than the sparse group.

This branch works okay locally on win32 and win64 so maybe the appveyor issue was spurious. Strange because win64 usually either works or times out, exiting during a linalg test without a specific message is not something I see very often.